cmd/bedMaxWig/bedMaxWig.go

```diff
@@ -12,14 +12,27 @@ import (
 	"log"
 )
 
-func bedMaxWig(infile string, database string, chromsizeFile string, outfile string) {
+func bedMaxWig(infile string, database string, chromsizeFile string, outfile string, norm bool) {
 	var records []bed.Bed = bed.Read(infile)
 	var wigData []wig.Wig = wig.Read(database)
 	var sizes []chromInfo.ChromInfo = chromInfo.ReadToSlice(chromsizeFile)
 	var outlist []bed.Bed
 	var currentBed bed.Bed = records[0]
 	var chromSlice []float64
 	var i int
+	var wigTotal float64 = 0
+
+	if norm == true { //TODO: This assumes fixedStep. Either should check for fixedStep or be able to handle both cases (fixedStep and variableStep)
```

Comment:
The code inside this "if" could have been a function that takes a slice of wiggles and gives back the sum of the values.  This might make the code a little bit easier to read, and also let people more easily re-use parts of your work